devel/spice-server/Portfile

```diff
@@ -18,41 +17,20 @@ homepage            https://www.spice-space.org/
 master_sites        https://www.spice-space.org/download/releases/
 use_bzip2           yes
 
-checksums           rmd160  fe99d8d0db4b275b1d19dc9a7231144133c5bfa0 \
-                    sha256  551d4be4a07667cf0543f3c895beb6da8a93ef5a9829f2ae47817be5e616a114 \
-                    size    1504304
-
-set py_ver          3.11
-set py_ver_nodot    [string map {. ""} $py_ver]
-
-patchfiles-append \
-                    meson-build.diff \
-                    no-werror.diff \
-                    pthread-setname-args.diff \
-                    no-msg-nosignal.diff \
-                    no-tools.diff \
-                    no-tests.diff \
-                    fix-py-module-search.diff \
-                    configurable-python.patch
-
-configure.args-append \
-    -Dsasl=false \
-    -Dpython=${prefix}/bin/python${py_ver}
-
-depends_build-append \
-    port:pkgconfig \
-    port:python${py_ver_nodot} \
-    port:py${py_ver_nodot}-six \
-    port:py${py_ver_nodot}-parsing
-
-depends_lib-append \
-    port:lz4 \
-    port:spice-protocol \
-    path:lib/pkgconfig/glib-2.0.pc:glib2 \
-    path:lib/pkgconfig/pixman-1.pc:libpixman \
-    path:include/turbojpeg.h:libjpeg-turbo \
-    port:gstreamer1 port:gstreamer1-gst-plugins-base \
-    port:libopus
+checksums           rmd160  a540a901e256ad2f806a25aa8861bcf7fdea507e \
+                    sha256  6d9eb6117f03917471c4bc10004abecff48a79fb85eb85a1c45f023377015b81 \
+                    size    1663326
+
+depends_build       port:pkgconfig
```

Comment:
Do prefer the `-append` suffix for the `depends_` statements:

- `depends_build-append`
- `depends_lib-append`

Reply:
It's ugly though. The meson port group is immune to this, it registers a callback.

Reply:
😆 Didn't think there would be an objection based on aesthetics :) It's more a best practice, it's not any kind of hard rule.

Reply:
Haha. Port groups should be transparent IMO, it should be handled at that level where possible.